src/index.html

```diff
@@ -1,17 +1,133 @@
 <!doctype html>
 <html lang="en">
+
   <head>
```

Comment:
Don't add empty lines between parent and child elements.
```suggestion
<html lang="en">
  <head>
```